### ISSUE TITLE
OCR: Position of OCR text off the mark and single page not OCRed at all

### ISSUE THREAD

**Issue by AEgit** [jbarlow83/OCRmyPDF#401]
**Describe the issue**
I tried to OCR the PDF provided below. The position of the OCR text from the actual (image) text in the PDF is quite far off. Furthermore, it seems that page 6 was not OCRed at all.
Note, that if I use the `--force` parameter, this issue seems to disappear (but I thought you might be interested in the file, to figure out, why "forcing" is necessary).

**To Reproduce**
The following command was used.

```bash
for f in *.pdf; do ocrmypdf -v -l eng ./"$f" ./"${f%.pdf}_ocr.pdf"; done
```

**Input file**
https://app.box.com/s/wp846p8z90834req6ejevbvj6m1uga9e

**Expected behavior**
OCR'ing without `--force` should be possible.

**System:**
OS: Ubuntu 18.04 (in Windows 10 WSL)
ocrmypdf 8.3.1
tesseract 4.0.0-beta.1
gs 9.26
qpdf 8.0.2

**Comment by jbarlow83** [comment]
PDF has a cropped mediabox that ends up being responsible for the offset. The images are shared among pages with each page display a portion of the image. The basic case is easy to fix but I need to check how it interacts with page rotation.

`--force` "burns in" the cropping so it fixes the alignment.

**Comment by AEgit** [comment]
Cheers, it is no major problem, since `--force` solves it, but I thought I should let you know about it.

**Comment by jbarlow83** [comment]
Fixed the basic case in v9.0.0.

**Comment by AEgit** [comment]
I can confirm this issue has been fixed in v9.0.0. In fact, it fixed another issue, that I was about to report as well. And I really like the newly introduced progress bars. Thank you very much!
